src/elements/content-explorer/stories/tests/ContentExplorer-visual.stories.js

```diff
@@ -42,36 +42,43 @@ export const openDeleteConfirmationDialog = {
     play: async ({ canvasElement }) => {
         const canvas = within(canvasElement);
 
-        const moreOptionsButton = await canvas.findByRole('button', { name: 'More options' });
-        await userEvent.click(moreOptionsButton);
+        const moreOptionsButton = await canvas.findAllByRole('button', { name: 'More options' });
```

Comment:
are there multiple buttons that show with the "More Options" button? if not why this change?

Reply:
there are multiple, have to indicator which one